### DISCUSSION TITLE
Powiadomianie o zanieczyszczeniu.

### DISCUSSION THREAD

**Discussion by skwaruch** [hackerspace-silesia/Smogomierz#141] [General]
Chciałbym otrzymywać powiadomienia o zanieczyszczeniu z mojego smogomierza. Znacie może jakieś sposoby darmowe? Czy AirMonitor ma jakąś aplikację na androida z powiadomieniami? Czy może sobie mail`a wysyłać?

**Comment by Eurynomosek** [comment]
jesli dodasz luftdaten to jest aplikacja kanarek i tam można śledzić rożne  czujniki
